test/fixtures/scenarios/makelogs/mapping.js

```diff
@@ -0,0 +1,110 @@
+module.exports = {
```

Comment:
This is more than a mapping, it's close to a template, but is really just an index create body